pkgs/applications/window-managers/picom/picom-next.nix

```diff
@@ -0,0 +1,13 @@
+{ picom }:
+
+picom.overrideAttrs (oldAttrs: rec {
+  pname = "picom-next";
+    version = "unstable-2021-11-01";
+    src = fetchFromGitHub {
+      owner = "yshui";
+      repo = "picom";
+      rev = "fade045eadf171d2c732820d6ebde7d1943a1397";
+      sha256 = "fPiLZ63+Bw5VCxVNqj9i5had2YLa+jFMMf85MYdqvHU=";
+      };
+    meta.maintainers = with maintainers; oldAttrs.meta.maintainers ++ [ GKasparov ];
+  })
```

Comment:
```suggestion
{ picom, lib, fetchFromGitHub }:

picom.overrideAttrs (oldAttrs: rec {
  pname = "picom-next";
  version = "unstable-2021-11-01";
  src = fetchFromGitHub {
    owner = "yshui";
    repo = "picom";
    rev = "fade045eadf171d2c732820d6ebde7d1943a1397";
    sha256 = "fPiLZ63+Bw5VCxVNqj9i5had2YLa+jFMMf85MYdqvHU=";
  };
  meta.maintainers = with lib.maintainers; oldAttrs.meta.maintainers ++ [ GKasparov ];
})
```

Reply:
Added lib import, changed to lib.maintainers and fixed indents